source/scss/_pf-basic-icons.scss

```diff
@@ -219,3 +219,12 @@
 .#{$pf-icon-prefix}-zone:before {
   content: $pficon-var-zone;
 }
+.pf-icon-circle-lg {
```

Comment:
I agree that circle is a modifier. And I wasn't sure about the use of double dashes, is that something I should include?

I like the idea of separating icon size from the circle. I think that would look like this: 

.pf-icon--circle {
  width: calc(2em + 4px);
  height: calc(2em + 4px);
  line-height: 2em;
  text-align: center;
  border: 2px solid $color-pf-blue-300;
  border-radius: 50%;
}
.pf-icon--lg {
  font-size: $icon-size-lg;
}
.pf-icon--md {
  font-size: $icon-size-md;
}

**_I don't think this is necessary:_**
.pf-icon--base {
  font-size: $font-size-relative-base;
}